SA/design_concept/content/poc/threepenny_gui.typ

```diff
@@ -0,0 +1,17 @@
+#import "../../../acronyms.typ": ac
+
+= Threepenny-GUI
```

Comment:
Maybe add something about how it's written in Haskell and allows interaction with the browser through a JavaScript FFI (foreign function interface)

Reply:
I added the "written in Haskell" part. I already mentioned the JS FFI, or should a write more detailed about that?

Reply:
oh, sorry I somehow didn't see that. Yes, in this case it's fine like it is 👍